network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -85,22 +76,34 @@ class PublicationPage(FoundationMetadataPageMixin, Page):
     )
     notes = RichTextField(
         blank=True,
+        features=['link', 'bold', 'italic']
     )
     contents_title = models.CharField(
         blank=True,
         default="Table of Contents",
         max_length=250,
     )
     content_panels = Page.content_panels + [
-        MultiFieldPanel([
-            FieldPanel('subtitle'),
-            FieldPanel('secondary_subtitle'),
-            FieldPanel('publication_date'),
-            ImageChooserPanel('hero_image'),
-            DocumentChooserPanel('publication_file'),
-            InlinePanel("authors", label="Author"),
-            FieldPanel("additional_author_copy"),
-        ], heading="Hero"),
-        FieldPanel('contents_title'),
-        FieldPanel('notes')
+        MultiFieldPanel(
+            [
+                FieldPanel("subtitle"),
+                FieldPanel("secondary_subtitle"),
+                FieldPanel("publication_date"),
+                ImageChooserPanel("hero_image"),
+                DocumentChooserPanel("publication_file"),
+                InlinePanel("authors", label="Author"),
+                FieldPanel("additional_author_copy"),
+            ],
+            heading="Hero",
+        ),
+        FieldPanel("contents_title"),
+        FieldPanel("notes"),
     ]
+
+    @property
+    def is_chapter_page(self) -> bool:
+        """Is this a chapter page (child-Publicationpage). Returns a bool."""
+        parent = self.get_parent().specific
+        return parent.__class__.__name__ == self.__class__.__name__
```

Comment:
Note that technically this code does not do what the property name promises: this code does not test whether something is a chapter page, it only tests whether it has the same class as its parent.  So at best it can conclude that something is a `PublicationPage`.

Is the logic that "a PublicationPage under a PublicationPage is informally called a chapter page"? If so, it might make more sense to explicitly compare the parent's `__class__`  to `PublicationPage` , given that, by definition, `self` is known to be a `PublicationPage`.